blogs/13_bfcl_v3_multi_turn.html

```diff
@@ -861,7 +856,8 @@ <h4>Response-based Evaluation and Its Limitations in Multi-Turn Evaluation</h4>
             through a trial-and-error process, it ultimately achieved the goal.</p>
           <p>This example illustrates how evaluating based solely on the response is not ideal. Sometimes, the model
             may not have enough information to determine the shortest trajectory, and taking extra steps should not be
-            unfairly penalized when the task is completed successfully.</p>
+            unfairly penalized when the task is completed successfully. Thus, we introduce a subset-matched response-based evaluation to address these limitations. The ground truth is the shortest trajectory that can achieve the task (e.g., minimial number of functions that must be called, even under the God's eye view of the initial configuration and avoid any exploration function calls). The model result is considered correct if it contains the ground truth as a subset, even if it contains additional function calls or takes a different trajectory.</p>
```

Comment:
s/e.g./i.e.

Reply:
I think `shortest trajectory that can achieve the task` may be confusing. Let's just rephrase this sentence into 

The ground truth is the minimial number of functions that must be called, under the God's eye view of API internal state changes, effectively avoid any exploration function calls.